src/bin/esyCreate.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @flow
+ */
+
+import type {CommandContext, CommandInvocation} from './esy';
+
+import chalk from 'chalk';
+import * as child from '@esy-ocaml/esy-install/src/util/child';
```

Comment:
nit: you can use `../lib/child_process.js`

Reply:
I can't remember why I opted to use it. I think I was trying to get rid of the stack trace and handle it in the template instead. Evidently, I failed. We should deal with it at some point later.

![image](https://user-images.githubusercontent.com/9691/34508336-0c51c228-f079-11e7-8e71-5710bf44935d.png)